tactics/equality.ml

```diff
@@ -479,6 +479,22 @@ let general_rewrite ~where:cls ~l2r:lft2rgt occs ~freeze:frzevars ~dep:dep_proof
             end
     end
 
+let clear_for_rewrite_in_hyps ids c =
+  let ids = Id.Set.of_list ids in
+  Proofview.Goal.enter begin fun gl ->
+    let env = Proofview.Goal.env gl in
+    let sigma = Proofview.Goal.sigma gl in
+    let is_section_variable = Termops.is_section_variable (Global.env()) in
```

Comment:
Use `tclENV` instead of `Global.env`.

Reply:
OK, I actually pondered a bit and changed my mind. Instead of exporting the `is_section_variable` in the Evarutil API, you should just export the function with the flag always set to the constant function true. It's a fast-path, and it's unlikely to matter here. While you're at it you could also not return the constr and pick a better name.